src/main/java/Controller/musicEngineController.java

```diff
@@ -64,7 +104,8 @@ private void stop(){
             case PLAYLIST:
```

Comment:
Case NONE and case SONG can be omitted entirely (or replaced with default: break; but its redundant)

Reply:
(unless you're adding to it actually. I'm seeing all the TODOs now, lol)